regnety/models/model.py

```diff
@@ -7,7 +7,7 @@
 
 
 def _get_model_with_config(config,  
-    userdef_input_shape: Union[List, Tuple] = (224,224,3)):
+    userdef_input_shape: Union[List, Tuple] = (320,320,3)):
```

Comment:
Are we training with 320x320 now? Or is there such that our input size becomes 320 -> random_crop -> resize to 224?

Reply:
That's correct. We have 512x512 images, we random crop them to 320x320 and pass them to the model. The PreStem layer resizes them to 224x224.